atoma-state/src/handlers.rs

```diff
@@ -16,6 +16,8 @@ use crate::{
     AtomaStateManager, AtomaStateManagerError,
 };
 
+const RATIO_FOR_CLAIM_STACK_THRESHOLD: f64 = 0.5;
```

Comment:
How was this ratio determined?

Reply:
The value should be `0.95`, it makes sense to claim 95% filled stacks.